### PR TITLE
Update kernel build instructions

### PR DESCRIPTION
**Pull request by pelwell**

### CODE REVIEW

**Review by aallan — APPROVED** [review]
YOLO

File: documentation/asciidoc/computers/linux_kernel/building.adoc [hunk 1]
```diff
@@ -62,7 +62,7 @@ KERNEL=kernel
 make bcmrpi_defconfig
 ----
 
-For Raspberry Pi 2, Pi 3, Pi 3+, and Compute Module 3 default build configuration
+For Raspberry PI Zero 2 W, Pi 2, Pi 3, Pi 3+, and Compute Module 3 default build configuration
```
Comment (lurch) [inline documentation/asciidoc/computers/linux_kernel/building.adoc:65]: @pelwell Should "Compute Module 3+" also be listed here and on line 189?

Comment (pelwell) [reply documentation/asciidoc/computers/linux_kernel/building.adoc:65]: CM3 can be considered to include CM3, but feel free to add it - and don't ask for a review.

Comment (lurch) [reply documentation/asciidoc/computers/linux_kernel/building.adoc:65]: Tidied up in #2428

**Pull request merged**